client/app/data/bwwc.js

```diff
@@ -575,6 +576,7 @@ define([], function () {
             },
             "type":"currency",
             "min":0,
+            "max_warning":100000000,
```

Comment:
Are these values from Shereen or are we just putting something that we think makes sense? 100 mil sounds like a lot but what do I know about CEO salaries.

Reply:
From Andrei in web-mpc slack:
"Shereen had inputs on what the maximum salary should be. She said she’ll look for the highest salary in Boston. We do want it to be lower than $120,000,000, according to her."

I put in $100,000,000 after looking up highest salaries in the US.

Reply:
I think it should be safe as an upper bound. Different sites disagree on who makes the most money in the US, and a few seem to make more than $100,000,000, but most of them don't since a lot of their money comes from other sources (e.g. bonuses, stock shares, blah blah blah).

Since I think the form is salary data not total compensation, we should be good?

Reply:
I think this is a safe upper bound---especially since this is a warning. Users can submit anyway if it ends up being higher.